app/paypal.py

```
@@ -0,0 +1,231 @@
+import re
+from pydantic import BaseModel, Field
```

Ruff (0.12.2)

`pydantic.Field` imported but unused

Remove unused import: `pydantic.Field`

(F401)